containers/podman/up.sh

```
@@ -394,37 +485,58 @@ print(yaml)
-# chown) or configure returns EACCES. Avoid world-writable chmod.
-ABBENAY_CONFIG_DIR="$ROOT/containers/abbenay/config"
-mkdir -p "$ABBENAY_CONFIG_DIR"
-# Relabel the empty dir first so seed copies land on a container-readable parent.
-_relabel_host_path_for_podman "$ABBENAY_CONFIG_DIR"
+# Never chown the git checkout: rootless Podman maps UID 1001 to a subordinate
+# host UID and would lock the developer out of containers/abbenay/config.
+# Runtime config always lives under CACHE_PATH (mode 0700/0600; credentials).
+ABBENAY_CONFIG_SEED="$ROOT/containers/abbenay/config"
+ABBENAY_CONFIG_REPO_PATH="$ABBENAY_CONFIG_SEED"
+ABBENAY_CONFIG_DIR="$CACHE_PATH/abbenay/config"
+mkdir -p -m 0700 "$ABBENAY_CONFIG_DIR"
```

Shellcheck (0.11.0)

[warning] When used with -p, -m only applies to the deepest directory.

(SC2174)